README.md

```diff
@@ -81,6 +81,7 @@ It is also important that your disks are setup according to the [SAP storage req
 |sap_hana_deployment_apply_license|Whether to apply a License File to the deployed HANA instance|no, defaulted to 'false'|
 |sap_hana_deployment_license_path|Target host directory path where HANA license file located|no, required if `sap_hana_deployment_apply_license` true|
 |sap_hana_deployment_license_file_name|HANA license file name|no, required if `sap_hana_deployment_apply_license` true|
+|sap_hana_deploymnet_skip| set this to 'true' if the role should not run the Hana Installer | no | 
```

Comment:
The goal of this role is to deploy SAP HANA and you are adding a variable to make this an option? This doesn't make any sense to me.

Reply:
Well this is just a side-effect. I need an internal variable to decide whether the installation command needs to be called or not. I thought it would be a good idea expose it, in case somebody is testing everything on smaller system but do not want to kick the actual installer. I can also hide it as an internal variable in var instead of defaults and prefix it with an underscore. Please let me know if you prefer this

Reply:
You don't need to define this variable in the inventory nor part of the role, as you are using it as Fact, dynamically adding it during execution time, so please remove any reference as this is confusing. Also, your supposition for a user defining the variable value for testing is wrong, since variables defined as part of the `set_fact` module will override any input from the user on the same variable defined as part of the role or the inventory, check [this](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#ansible-variable-precedence) for your understanding.

Please remove the entry from README and defaults/main files.

Reply:
DONE